mongodbatlas/resource_mongodbatlas_cloud_provider_snapshot_restore_job.go

```diff
@@ -331,7 +331,7 @@ func resourceMongoDBAtlasCloudProviderSnapshotRestoreJobImportState(d *schema.Re
 
 	d.SetId(encodeStateID(map[string]string{
 		"project_id":              *projectID,
-		"cluster_name":            *clusterName,
+		"cluster_name":            requestParameters.ClusterName,
```

Comment:
I asked about similar fixes in the other pull requests.  If we fix a bug while fixing the tests that should be noted in the PR, that way users know what was fixed.  It's just a "nice to do" and makes understanding what happened easier.

Reply:
This is an import that sometimes passes or fails, need to see why does happen, not sure if it's because there are many test acceptances running at same time. I changed this to verify it was the issue but it isn't, that's why I added a env var to skip test import if true in all test acceptance that imports because if you skip one another can appear with similar error.

Reply:
Okay then if the change didn't fix anything and is not needed it shouldn't be included unless it's there for a reason since this isn't just the test but the actual resource code.

Reply:
Ok, I'll change it back for now, thank you